lib/rubocop/cop/lint/redundant_block_reference_argument.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module RuboCop
+  module Cop
+    module Lint
+      # This cop checks for a redundant block reference argument.
+      # Such arguments should be omitted.
+      #
+      # @example
+      #
+      #   @bad
+      #   def foo(&_)
+      #   end
+      #
+      #   @good
+      #   def foo
+      #   end
+      class RedundantBlockReferenceArgument < Cop
+        MSG = 'Found redundant block reference argument. Omit it.'.freeze
```

Comment:
I'd prefer this to be a single sentence mentioning the problematic argument explicitly (some people might not understand the term used here).